src/language/interfaces/commands.ts

```diff
@@ -0,0 +1,75 @@
+/*---------------------------------------------------------------------------------------------
+ *  Copyright (c) [2017] [Vincas Stonys].
```

Comment:
@rthomas320 @michael-hoke Would be best to answer this as they really spearheaded this part.

Reply:
@mbeckerle According to @rthomas320 @michael-hoke Regis added the license to every file from that extension so that they all matched - there wasn't a consistent license header across all files in that project.